src/components/MewSwitch/MewSwitch.vue

```diff
@@ -18,20 +17,14 @@
 
 export default {
   name: 'MewSwitch',
-  data() {
-    return {
-      isSwitchOn: false
-    }
-  },
   props: {
     label: {
       type: String,
       default: ''
-    }
-  },
-  methods: {
-    onToggle() {
-      this.isSwitchOn = !this.isSwitchOn;
+    },
+    switchStatus: {
```

Comment:
I totally forgot to write the comments for label for this above  , could you add this on top of label ? 
    /**
     * The switch label.
     */ 

and just add the one for switchStatus!
thank you!!